web-server/src/slices/org.ts

```diff
@@ -194,3 +217,39 @@ export const updatedOrgAlertSettings = createAsyncThunk(
     );
   }
 );
+
+export const getDefaultSyncDaysSettings = createAsyncThunk(
+  'org/getDefaultSyncDaysSettings',
+  async (params: { orgId: ID }) => {
+    return (
+      await handleApi<OrgDefaultSyncDaysSettings>(
+        `/internal/${params.orgId}/settings`,
+        {
+          params: {
+            setting_type: 'DEFAULT_SYNC_DAYS_SETTING'
+          }
+        }
+      )
+    ).default_sync_days;
```

Comment:
This is async method, so I used await.

Reply:
I think the `.default_sync_days` property read is only going to happen if the promise resolves successfully, in which case there's functionally no difference between .then or just (await handleApi(...)).default_sync_days.

So this may be a change that can go unchanged.